tests/test_custom_models.py

```diff
@@ -689,6 +703,8 @@ def test_merge_layers(self, test_name, model_id, config_cls, config_kwargs):
             config_kwargs["init_ia3_weights"] = False
         elif issubclass(config_cls, LNTuningConfig):
             pass
+        elif issubclass(config_cls, FourierConfig):
```

Comment:
This check can be merged with the previous: `elif issubclass(config_cls, LNTuningConfig, FourierConfig):`